src/StatusBarItem.ts

```diff
@@ -37,9 +37,14 @@ export class StatusBarItem extends vscode.Disposable {
   ) {
     this.item.command = Command.StopWatching
     const total = passed + failed
+    const percentOfExecutedTests = Number((passed / total * 100).toFixed(0))
+    const percentIsValid = Boolean(percentOfExecutedTests)
```

Comment:
percentOfExecutedTests might be 0, and this will be false

Reply:
I used a boolean because if `percentOfExecutedTests` is 0(falsy) the percent number receives a 0 of the ternary:
`const percentNumber = percentIsValid ? percentOfExecutedTests : 0﻿﻿`

However, make more sense to use an `isNaN` instead `Boolean` :) Thanks for the review!